src/index.js

```diff
@@ -58,9 +58,9 @@ function getExampleFromParamObject(param_obj, getExampleFromType) {
   return getExampleFromType({ type: param_obj.type, format: param_obj.format, enum: param_obj.enum });
 }
 
-function constructRequest(unexpanded_path, method, operation_object, getExampleFromType, base_path) {
+function constructRequest(original_path, method, operation_object, getExampleFromType, base_path) {
   const path_params = operation_object.parameters ? operation_object.parameters.filter(p => p.in === 'path') : [];
-  let path = unexpanded_path;
+  let path = original_path;
```

Comment:
Hmm... does this actually create a new string? Your test implies this works, but I don't get how as you mutate the path in the for loop.